chain/messagepool/repub.go

```diff
@@ -0,0 +1,106 @@
+package messagepool
+
+import (
+	"context"
+	"sort"
+
+	"golang.org/x/xerrors"
+
+	"github.com/filecoin-project/go-address"
+	"github.com/filecoin-project/lotus/build"
+	"github.com/filecoin-project/lotus/chain/types"
+)
+
+const repubMsgLimit = 5
```

Comment:
Can we increase this to like 25-30?

Reply:
Given that we also republish less than once in 5 blocks.

Reply:
absolutely, I was also thinking that 5 is ridiculously low.

Reply:
done, set it to 30.